### DISCUSSION TITLE
csaf-poc/csaf_distribution: repo moved

### DISCUSSION THREAD

**Discussion by tschmidtb51** [aquasecurity/trivy#7804] [Development]
For your awareness:

As the tools are not a PoC (even since the first release), the long overdue change was conducted: The repo https://github.com/csaf-poc/csaf_distribution moved to https://github.com/gocsaf/csaf. The old URL can still be used for a couple month before it is sunsetted for security reasons.
Also, the license changed from MIT to Apache 2.0 (on the main branch, there is no new release yet).

Currently, that is mentioned in 
https://github.com/aquasecurity/trivy/blob/63dd3d65a32296eee529c41c41f3221e2cd6d3fe/go.mod#L45

Please change the URL at an appropriate time.

**Comment by knqyf263** [comment]
Thanks https://github.com/aquasecurity/trivy/issues/7805
